x-pack/plugins/uptime/public/components/monitor/ping_list/ping_list.tsx

```diff
@@ -54,6 +55,12 @@ export const PingList = () => {
     Object.keys(expandedRows).filter((e) => !pings.some(({ docId }) => docId === e))
   );
 
+  useEffect(() => {
```

Comment:
Nice simple strategy here, small footprint, runs once. I smoke tested this and it seemed to work great.